src/ocf_seq_cutoff.c

```diff
@@ -62,6 +63,27 @@ void ocf_core_seq_cutoff_init(ocf_core_t core)
 	}
 }
 
+void ocf_dbg_get_seq_cutoff_status(ocf_core_t core,
+		struct ocf_dbg_seq_cutoff_status *status)
+{
+	uint32_t threshold = ocf_core_get_seq_cutoff_threshold(core);
+	struct ocf_seq_cutoff_stream *stream;
+	int i = 0;
+
+	OCF_CHECK_NULL(core);
```

Comment:
null check after touching the pointer at line 69

Reply:
Fixed!